src/sentry/monitors/validators.py

```diff
@@ -230,21 +230,25 @@ def validate(self, attrs):
         return attrs
 
 
-@extend_schema_serializer(exclude_fields=["project", "name", "slug", "config", "alert_rule"])
+@extend_schema_serializer(exclude_fields=["project", "config", "alert_rule"])
 class MonitorValidator(CamelSnakeSerializer, PreventNumericSlugMixin):
     project = ProjectField(scope="project:read")
-    name = serializers.CharField(max_length=128)
+    name = serializers.CharField(
+        max_length=128, help_text="Name of the monitor. Used for notifications."
```

Comment:
nit: Let's slap a [magic trailing comma](https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#the-magic-trailing-comma) at the end so it's consistent with the other fields?